lib/devise_two_factor/spec_helpers/two_factor_authenticatable_shared_examples.rb

```diff
@@ -102,8 +102,8 @@
     end
 
     it 'should return uri with issuer option' do
-      expect(subject.otp_provisioning_uri(account, issuer: issuer)).to match(%r{otpauth://totp/#{account}\?.*secret=\w{#{otp_secret_length}}(&|$)})
-      expect(subject.otp_provisioning_uri(account, issuer: issuer)).to match(%r{otpauth://totp/#{account}\?.*issuer=#{issuer}(&|$)})
+      expect(subject.otp_provisioning_uri(account, issuer: issuer)).to match(%r{otpauth://totp/#{issuer}:#{account}\?.*secret=\w{#{otp_secret_length}}(&|$)})
```

Comment:
[Here's](https://github.com/mdp/rotp/commit/07825ecdcdaf9db7b6357b002f65f538ad4634bd#diff-ed386b7f6dadc0ef09bf81c543cffbfbR61) where ROTP starts including the issuer string in the URI.